includes/namespace.php

```diff
@@ -29,63 +29,111 @@
 use WP_Error;
 use WP_Site;
 
-if (
-	! class_exists( '\Google\Web_Stories\Plugin' ) ||
-	! file_exists( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/js/edit-story.js' )
-) {
-	/**
-	 * Displays an admin notice about why the plugin is unable to load.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @return void
-	 */
-	function _print_missing_build_admin_notice() {
-		?>
-		<div class="notice notice-error">
-			<p>
-				<strong><?php esc_html_e( 'Web Stories plugin could not be initialized.', 'web-stories' ); ?></strong>
-			</p>
-			<p>
-				<?php
-					echo wp_kses(
-						sprintf(
-						/* translators: %s: build commands. */
-							__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ),
-							'<code>composer install &amp;&amp; npm install &amp;&amp; npm run build</code>'
-						),
-						[
-							'code' => [],
-						]
-					);
-				?>
-			</p>
-		</div>
-		<?php
-	}
+// Load Compatibility class the old fashioned way.
+if ( ! class_exists( 'Compatibility' ) ) {
```

Comment:
I believe this class name needs to be fully-qualified, so:

```suggestion
if ( ! class_exists( 'Google\Web_Stories\Compatibility' ) ) {
```

Reply:
```suggestion
if ( ! class_exists( '\Google\Web_Stories\Compatibility' ) ) {
```